src/cbmc/cbmc_parse_options.cpp

```diff
@@ -553,6 +546,26 @@ int cbmc_parse_optionst::doit()
   if(set_properties())
     return CPROVER_EXIT_SET_PROPERTIES_FAILED;
 
+  // At this point, our goto-model should be in core-goto form (all of the
+  // transformations have been run and the program is ready to be given to the
+  // solver).
+  if(cmdline.isset("export-core-goto"))
+  {
+    auto core_goto_filename = cmdline.get_value("export-core-goto");
```

Comment:
:grey_question: What happens if the user specifies `--export-core-goto` without specifying an output filename?

Reply:
I have added extra checks for a missing filename.